backend/src/Controller/CaptainFinancialSystem/CaptainFinancialSystemAccordingToCountOfHoursController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Controller\CaptainFinancialSystem;
+
+use App\AutoMapping;
+use App\Controller\BaseController;
+use App\Service\CaptainFinancialSystem\CaptainFinancialSystemAccordingToCountOfHoursService;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use OpenApi\Annotations as OA;
+
+/**
+ * Create and fetch Captain's Financial System According To Count Of Hours.
+ * @Route("v1/captainfinancialsystemaccordingtocountofhours/")
+ */
+class CaptainFinancialSystemAccordingToCountOfHoursController extends BaseController
+{
+    private CaptainFinancialSystemAccordingToCountOfHoursService $CaptainFinancialSystemAccordingToCountOfHoursService;
```

Comment:
Change the variable name (can not start with capital letter), and check all places in which it is being used.